types/src/common_params.rs

```diff
@@ -206,6 +206,48 @@ impl From<Params> for CommonParams {
     }
 }
 
+impl Into<Params> for CommonParams {
```

Comment:
Why do you implement `Into` instead of `From<CommonParams> for Params`?

Reply:
FYI, there is only one case that you need to implement `Into` instead of `From`; the input of the conversion is a generic type that is declared in the other crate.